db/migrate/20221108064710_add_postalcode_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostalcodeToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :postalcode, :string
```

Comment:
```suggestion
    add_column :users, :postal_code, :string
```

2単語なので`_`を挟んだ方がベターです